lib/trento_web/channels/user_channel.ex

```diff
@@ -0,0 +1,34 @@
+defmodule TrentoWeb.UserChannel do
+  @moduledoc """
+  User channel, each use is subscribed to his channel,
+  to receive personal broadcasts
+
+  Users can't join other users channels.
+  This has as prerequisite the websocket authentication defined in user socket
+  """
+  require Logger
+  use TrentoWeb, :channel
+
+  @impl true
+  def join(
+        "users:" <> user_id,
+        _payload,
+        %{assigns: %{current_user_id: current_user_id}} = socket
+      ) do
+    user_id = String.to_integer(user_id)
```

Comment:
General question why are we using integer ID and not UUID ?
I guess to keep it simple ?